c/blake3_llfio.cpp

```diff
@@ -0,0 +1,103 @@
+#include <variant>
+
+#include <llfio.hpp>
+
+#include "blake3.h"
+#include "blake3_impl.h"
+
+namespace llfio = LLFIO_V2_NAMESPACE;
+
+template <class... Ts> struct overloads : Ts... {
+  using Ts::operator()...;
+};
+// NOTE: deduction guide only needed for Apple Clang now.
+template <class... Ts> overloads(Ts...) -> overloads<Ts...>;
+
+INLINE auto copy_wide(blake3_hasher *self, llfio::path_view path,
+                      llfio::file_handle &file) noexcept
+    -> llfio::result<std::uint64_t> {
+  std::array<std::byte, 65535> buffer{};
+  std::uint64_t total = 0;
```

Comment:
I don't think we use the value of `total` anywhere, right?

Reply:
We use it here: https://github.com/BLAKE3-team/BLAKE3/blob/fac71c9e6748e61e0c280e4e905f8f808b3c2ae8/c/blake3_llfio.cpp#L30

and here:
https://github.com/BLAKE3-team/BLAKE3/blob/fac71c9e6748e61e0c280e4e905f8f808b3c2ae8/c/blake3_llfio.cpp#L36

Reply:
yes, I should've elaborated more. We don't do anything with the return value of `copy_wide` at the call-sites and therefore we do nothing with the `total` value AFAICT

Reply:
I wrote it that way because I was trying to follow the structure of the definitions in [`io.rs`](https://github.com/BLAKE3-team/BLAKE3/blob/master/src/io.rs) and there `copy_wide` is written to return `total`.

But you're right it isn't actually used for anything here nor is it actually used in equivalent the Rust code.

I could remove it.